src/main/java/com/javadiscord/javabot/events/InteractionListener.java

```diff
@@ -26,10 +28,11 @@ public void onButtonClick(ButtonClickEvent event) {
 		Guild guild = preferredGuild;
 		MongoDatabase database = mongoClient.getDatabase("other");
 		String[] id = event.getComponentId().split(":");
+		System.out.println(event.getComponentId());
 		switch (id[0]) {
 			case "dm-submission" -> this.handleDmSubmission(database, guild, event);
 			case "submission" -> this.handleSubmission(database, guild, event);
-			case "reactionroles" -> this.handleReactionRoles(database, guild, event);
+			case "reaction-role" -> this.handleReactionRoles(event);
 		}
```

Comment:
We should really create an actual structure for all button interactions, instead of just adding hard-coded things to a switch-case statement.